web/rootfs/defaults/meet.conf

```diff
@@ -47,16 +47,37 @@ location @root_path {
 
 {{ if .Env.ETHERPAD_URL_BASE }}
 # Etherpad-lite
-location /etherpad/ {
-    proxy_http_version 1.1;
-    proxy_set_header Upgrade $http_upgrade;
-    proxy_set_header Connection 'upgrade';
-    proxy_set_header Host $host;
-    proxy_cache_bypass $http_upgrade;
-
-    proxy_pass {{ .Env.ETHERPAD_URL_BASE }}/;
-    proxy_set_header X-Forwarded-For $remote_addr;
-    proxy_buffering off;
-    proxy_set_header Host {{ .Env.XMPP_DOMAIN }};
-}
+location ^~ /etherpad {
+        rewrite                /etherpad/(.*) /$1 break;
+        rewrite                ^/etherpad$ /etherpad/ permanent;
+        proxy_pass             {{ .Env.ETHERPAD_URL_BASE }}/;
+        proxy_pass_header Server;
+        proxy_redirect         / /etherpad/;
+        proxy_set_header       Host {{ .Env.XMPP_DOMAIN }};
+        proxy_buffering off;
+    }
+
+location ^~ /etherpad/socket.io {
+        rewrite /etherpad/socket.io/(.*) /socket.io/$1 break;
+        proxy_pass {{ .Env.ETHERPAD_URL_BASE }}/;
+        proxy_redirect         / /etherpad/;
+        proxy_set_header Host {{ .Env.XMPP_DOMAIN }};
+        proxy_buffering off;
+        proxy_set_header X-Real-IP $remote_addr;  # http://wiki.nginx.org/HttpProxyModule
+        proxy_set_header X-Forwarded-For $remote_addr; # EP logs to show the actual remote IP
+        proxy_set_header X-Forwarded-Proto $scheme; # for EP to set secure cookie flag when https is used
+        proxy_set_header Host {{ .Env.XMPP_DOMAIN }};  # pass the host header
+        proxy_http_version 1.1;  # recommended with keepalive connections
+        # WebSocket proxying - from http://nginx.org/en/docs/http/websocket.html
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection 'upgrade';
+    }
+
+location ^~ /static {
```

Comment:
Won't this break the static assets of jitsi meet?

Reply:
i thought so too but i couldn't find any issues so far